src/Symfony/Component/DependencyInjection/Extension/ExtensionInterface.php

```diff
@@ -37,7 +37,7 @@ public function getNamespace();
     /**
      * Returns the base path for the XSD files.
      *
-     * @return string The XSD base path
+     * @return string|false
```

Comment:
😢

Reply:
let's fix that with a deprecation, same as the guard method